AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/Mutations.dfy

```diff
@@ -463,4 +467,129 @@ module {:options "/functionSyntax:4" } Mutations {
     );
     return Success(mutatedItem);
   }
+
+  method MutateToHV2(
```

Comment:
To mutate we do:
- Decrypt the HV-1 item
- Modify the Branch Key Context (BKC) according to the Mutation (EC, KMS-ARN, HV)
- Calculate the BKC Digest with the modified BKC
- Pack and Encrypt the Branch Key Item